### ISSUE TITLE
Fleshing out /// notation

### ISSUE THREAD

**Issue by rmbarnett-rice** [capergroup/bayou#59]
In our current Evidence api call notation examples we show the user that:

```
{ 
    Evidence.apicalls("getInputStream");
    Evidence.types("BluetoothSocket");
}
```

Denotes something akin to"call getInputStream() on an instance of BluetoothSocket"

As such, when thinking about a corresponding /// notation do we want something verbatim like:

`/// calls:getInputStream types:BluetoothSocket`

Or something that is more reflective of the intended structure like:

`/// BluetoothSocket.getInputStream()`

**Comment by swaratchaudhuri** [comment]
The former is better:
/// calls:getInputStream types:BluetoothSocket

In fact, we should even permit:
/// getInputStream BluetoothSocket

**Comment by vijay-murali** [comment]
I agree, the former is preferred. This is because it's not always the case that the "type" and "API call" are from the same class, in which case type.APIcall() would not make sense.

**Comment by rmbarnett-rice** [comment]
Ok, here are the tests we are passing now based on the rewrites of the example .java files.  Let's confirm this is what we want.

https://gist.github.com/rmbarnett-rice/85b58178a22c071815d81675632bd965

**Comment by vijay-murali** [comment]
Support for /// has been added and merged with master in #61. I'm closing this thread, but feel free to re-open for further discussion.
